src/forges/Bitbucket/repositories.jl

```diff
@@ -61,7 +61,8 @@ into(::BitbucketAPI, ::typeof(create_repo)) = Repo
 function is_bitbucket_collaborator(resp::HTTP.Response)
     if HTTP.status(resp) == 200
         d = JSON2.read(String(HTTP.body(resp)), Dict)
-        return haskey(d, "values") && length(d["values"]) > 0
+        return haskey(d, "values") && length(d["values"]) > 0 &&
+            d["values"][1]["permissions"] in ["write" "admin"]
```

Comment:
Do we need to check `haskey(d["values"][1], "permissions")` as well? Also comma is missing between `"write"` and `"admin"`.

Reply:
We don't need haskey...permissions because if there are values, they will have permissions.

The comma isn't needed in the array.